app/views/coronavirus_landing_page/show.html.erb

```diff
@@ -1,21 +1,42 @@
 <%=
   render(
-      partial: 'page_header', locals: { breadcrumbs: breadcrumbs }
+    partial: 'page_header', locals: { breadcrumbs: breadcrumbs }
   )
 %>
 
 <div class="govuk-width-container">
-  <%= render 'components/header-notice', {
-    nhs_branding: true,
-    title_logo_url: image_url('nhs-logotype-blue.svg'),
-    heading: t("coronavirus_landing_page.nhs_banner.heading"),
-    list: t("coronavirus_landing_page.nhs_banner.list"),
-    call_to_action: {
-      href: t("coronavirus_landing_page.nhs_banner.call_to_action.href"),
-      title: t("coronavirus_landing_page.nhs_banner.call_to_action.title")
-    }
-  } %>
-  <%= render partial: 'explainer' %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-third">
+      <%= render 'components/header-notice', {
+        nhs_branding: true,
+        title_logo_url: image_url('nhs-logotype-blue.svg'),
+        heading: t("coronavirus_landing_page.nhs_banner.heading"),
+        list: t("coronavirus_landing_page.nhs_banner.list"),
+        call_to_action: {
+          href: t("coronavirus_landing_page.nhs_banner.call_to_action.href"),
+          title: t("coronavirus_landing_page.nhs_banner.call_to_action.title")
+        }
+      } %>
+    </div>
 
-  <%= render partial: 'accordion_sections'%>
+    <div class="govuk-grid-column-two-thirds">
+      <%= render partial: 'accordion_sections'%>
+      <%= render partial: 'topic_section' %>
+
+      <div class="landing-page__email-wrapper">
+        <svg  class="app-c-email-link__icon" xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 459.334 459.334">
+          <path fill="black" d="M177.216 404.514c-.001.12-.009.239-.009.359 0 30.078 24.383 54.461 54.461 54.461s54.461-24.383 54.461-54.461c0-.12-.008-.239-.009-.359H175.216zM403.549 336.438l-49.015-72.002v-89.83c0-60.581-43.144-111.079-100.381-122.459V24.485C254.152 10.963 243.19 0 229.667 0s-24.485 10.963-24.485 24.485v27.663c-57.237 11.381-100.381 61.879-100.381 122.459v89.83l-49.015 72.002a24.76 24.76 0 0 0 20.468 38.693H383.08a24.761 24.761 0 0 0 20.469-38.694z"/>
+        </svg>
+        <h3 class="govuk-heading-s landing-page__email-title"><%= t("coronavirus_landing_page.notifications.intro") %></h3>
+        <%= link_to(t("coronavirus_landing_page.notifications.email_link"), "/email-signup/?topic=", {
+          class: "govuk-body govuk-link",
+          data: {
+            module: "track-click",
+            "track-category": "",
```

Comment:
These are empty?

Reply:
Yeah, see the commit message and the slack chat.

Reply:
Now filled.